lib/generators/active_admin/install/templates/migrations/create_active_admin_comments.rb

```diff
@@ -9,7 +9,11 @@ def self.up
       t.string :resource_id,   null: false
       t.string :resource_type, null: false
       t.references :author, polymorphic: true
-      t.timestamps
+      if Rails::VERSION::MAJOR >= 5
+        t.timestamps
+      else
+        t.timestamps null: false
```

Comment:
Absolutely. The only advantage I see with the current version is that the version check can act as a reminder to leave the simpler `t.timestamps` once Rails 4.2 support is dropped.